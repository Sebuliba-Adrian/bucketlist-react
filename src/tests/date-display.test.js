import React from 'react';
import renderer from 'react-test-renderer';
import DateDisplay from '../components/date-display';

const props = {
  date: 'Wed, 20 Sep 2017 11:31:50 GMT',
};

it('renders correctly', () => {
  const dateDisplay = renderer.create(<DateDisplay {...props} />).toJSON();
  expect(dateDisplay).toMatchSnapshot();
});
