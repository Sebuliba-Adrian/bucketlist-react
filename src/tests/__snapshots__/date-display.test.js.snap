// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="text-center date-display"
>
  <div
    className="date-display-month text-white bg-cool-blue"
  >
     Sep
  </div>
  <div
    className="date-display-day"
  >
    <h4>
       20
    </h4>
  </div>
  <div
    className="date-display-year"
  >
     2017
  </div>
</div>
`;
